src/woo_publications/api/models/category.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from ordered_model.models import OrderedModel
+
+from ..constants import InformatieCategorieChoices
+
+
+class InformatieCategorie(OrderedModel):
+    identifier = models.URLField(_("Identifier"), max_length=255, unique=True)
+    name = models.CharField(_("Name"), max_length=80)
+    name_plural = models.CharField(_("Name plural"), max_length=80)
```

Comment:
```suggestion
    name_plural = models.CharField(_("Name plural"), max_length=80, blank=True)
```
I thought you mentioned yesterday that this is not guaranteed to be provided